### DISCUSSION TITLE
In Detailed Routing Site not found for [ERROR DPL-4219] Cell peripheral_subsystem_i/gpio_i/i_reg_file/2301 has no site.

### DISCUSSION THREAD

**Discussion by Wiki1998-dev** [The-OpenROAD-Project/OpenROAD-flow-scripts#2292] [Q&A] [answered]
What should i do please guide 
in the step of 3.5 detailed placement i am getting this error i have shared link which includes this log also in directory
This program is licensed under the BSD-3 license. See the LICENSE file for details.
Components of this program may be licensed under more restrictive licenses which must be honored.
[WARNING STA-1173] ./lib/sky130_fd_sc_lp__ss_150C_1v65.lib line 17, default_fanout_load is 0.0.
[ERROR DPL-4219] Cell peripheral_subsystem_i/gpio_i/i_reg_file/2301 has no site.
Error: detail_place.tcl, 10 DPL-4219
Command exited with non-zero status 1
Elapsed time: 0:21.73[h:]min:sec. CPU time: user 17.53 sys 2.44 (91%). Peak memory: 5906224KB.
make: *** [Makefile:544: results/sky130hd/core_v_mini_mcu/base/3_5_place_dp.odb] Error 1
I tried to increase area limitations by modifying to
export CORE_UTILIZATION = 5 # Very low utilization for maximum area
export PLACE_DENSITY = 0.05 # Very low placement density
export CORE_MARGIN = 10 # Large margin around the core
but still ..
can you please guide me why ad what is this issue about ?
Thanks
the design is core-v-mini-mcu
and
the link of all the files of project is 
https://drive.google.com/drive/folders/1Xh2Xdn4mJMHYTmsGoT-mEJkIJxp52Np3?usp=sharing

**Comment by maliberty** [comment, marked as answer]
You can use 'make issue' to package test cases rather than upload all of ORFS.  I downloaded your repo and found it fails in synthesis as written due to sky130_sram_4kbyte_1rw not being in ADDITIONAL_(LEF|LIB).  When I fixed that I got past dpl.  It is inefficient try to reproduce your problem this way.

This would have been better filed as an issue rather than a discussion.

**Comment by Wiki1998-dev** [comment]
Thanks a lot i am updating mz version of ORFS maybe problem lies there .
Sorry for trouble, next time i will keep in mind , to make it right way
